src/validationinterface.cpp

```diff
@@ -12,7 +12,7 @@
 #include <primitives/block.h>
 #include <primitives/transaction.h>
 #include <scheduler.h>
-
+#include <evo/deterministicmns.h>
```

Comment:
you don't need this `<include>` here probably, seems as just forward declaration is enough

Reply:
```
validationinterface.cpp:291:25: error: by-copy capture of variable 'tip_mn_list' with incomplete type 'const CDeterministicMNList'
    auto event = [vote, tip_mn_list, this] {
                        ^
./validationinterface.h:23:7: note: forward declaration of 'CDeterministicMNList'
class CDeterministicMNList;
      ^
```

Reply:
can you capture here by reference instead?

Reply:
ping: 1e0bb3b7ea0022e308beb50bf01f384731d27de3